composer.json

```diff
@@ -132,6 +134,10 @@
             "Composer\\Config::disableProcessTimeout",
             "paraunit run --testsuite integration"
         ],
+        "test:mutation": [
+            "Composer\\Config::disableProcessTimeout",
+            "infection --threads=max --only-covered --min-covered-msi=80"
```

Comment:
`--only-covered --min-covered-msi` can go in `infection.json5.dist`. good to have config in one place?

Reply:
`--min-covered-msi` can't be defined in the dist file because it's used both for Composer script (full analysis) and for CI, where [we don't require min MSI for PRs](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7874#discussion_r1533414437). `infection.json5.dist` must contain only truly common stuff, `--min-covered-msi` is contextual in our case.